material-overrides/assets/stylesheets/moonbeam.css

```diff
@@ -2014,3 +2015,143 @@ p.quick-ref-title {
   height: unset;
   padding: 1em 2em;
 }
+
+/* Cookie Consent Styling */
+.md-consent__inner {
+  z-index: 999;
+}
+
+form.md-consent__form.md-grid.md-typeset {
+  border: var(--border-width-thin) solid var(--light-transparent-15);
+  display: flex;
+  align-items: center;
+  padding: 1.5rem;
+  border-radius: 8px;
+  max-width: 90%;
+  margin: 2em auto;
+}
+
+/* Logo Section */
+.cookie-logo img {
+  display: flex;
+  margin: 2em;
+  width: 50px;
+}
+
+/* Content Section */
+.cookie-content {
+  padding: 1em;
+  flex: 1;
+  margin-right: 2em;
+}
+
+.cookie-content h4 {
+  margin: 1em 1em auto;
+  font-size: 15px;
+}
+
+.cookie-content p {
+  margin: 0 1em auto;
+  font-size: 15px;
+  color: var(--light-transparent-60);
+}
```

Comment:
you're setting left and right margin properties on these two items plus the `.md-consent__settings` instead of applying this margin to the parent property, which will then be applied to these elements automatically.

It's easier to maintain when we set it in one place. You might still need to control the top and bottom margins for each individual element here, because that's not as straightforward when you've got different element types like an h4, p, div, etc.